library/oneview_id_pools.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2021) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+ANSIBLE_METADATA = {'status': ['stableinterface'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: oneview_id_pools
+short_description: Manage OneView Id Pools.
+description:
+    - Provides an interface to manage Id pools. Can retrieve, update.
+version_added: "2.4"
+requirements:
+    - "python >= 3.4.2"
+    - "hpeOneView >= 5.6.0"
```

Comment:
since we refactored this resource in python hpeOneView >= 6.0.0, use the same here